src/plugin.json

```
@@ -9,7 +9,7 @@
   "backend": true,
   "executable": "doitintl-bigquery-datasource",
   "dependencies": {
-    "grafanaDependency": "7.x.x"
+    "grafanaDependency": ">=7.0.0"
```

I'd recommend a slightly more conservative approach:
```suggestion
    "grafanaDependency": "7.x || 8.x"
```
(assuming the release does in fact work on known 7x and 8.x releases)